Sources/Helpers/OASettingsHelper.h

```diff
@@ -63,6 +63,24 @@ typedef enum : NSUInteger {
 
 @end
 
+typedef enum : NSInteger {
```

Comment:
Always use NS_ENUM instead of the C enum! NS_ENUM is compatible with Swift and is used in every other place in the project.